addons/base_automation/migrations/11.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+model_renames_base_action_rule = [
+    ('base.action.rule', 'base.automation')
+]
+
+table_renames_base_action_rule = [
+    ('base_action_rule', 'base_automation')
+]
```

Comment:
@StefanRijnhart 
are model and table rename deprecated by apriori ?

https://github.com/OCA/OpenUpgrade/blob/11.0/odoo/addons/openupgrade_records/lib/apriori.py#L28

Reply:
No, we leave the renames to the originating modules. Apriori is to improve the analysis.

Reply:
Ok